massa-grpc/src/config.rs

```diff
@@ -90,4 +92,10 @@ pub struct GrpcConfig {
     pub max_block_ids_per_request: u32,
     /// max number of operation ids that can be included in a single request
     pub max_operation_ids_per_request: u32,
+    /// server certificate path
+    pub server_certificate_path: PathBuf,
```

Comment:
PathBuf could be Option<PathBuf>  ? if enable_mtls = false

Reply:
I tried to copy existing configuration style... we could change this in the future